offices/students-2/index.html

```diff
@@ -1,11 +1,10 @@
 
 {% set query = queries.office %}
-{% set results = query.search_with_url_arguments(q="_id:students-2",size=1) %}
-{% set office = results | first %}
+{% set office = get_document("office", "students-2") %}
 {% set path = '/offices/' + office.slug + '/' %}
 {% set resource_title = "Student resources" %}
 {% set activity_type = "Students" %}
 
-{% set initiatives = queries.initiative.search_with_url_arguments(q="related_office:students-2") %}
+{% set initiatives = queries.initiative.search_with_url_arguments(filter_related_office="students-2") %}
```

Comment:
@jimmynotjim: @dpford can correct me if I'm wrong, but I'm pretty sure you can filter on any field in Elasticsearch. This particular change creates a [term filter](http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/query-dsl-term-filter.html) for `students-2` in the field `related_office`.